src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/domain/UnallocatedCaseRisks.kt

```diff
@@ -17,7 +17,7 @@ data class UnallocatedCaseRisks @JsonCreator constructor(
   val tier: String,
   val activeRegistrations: List<UnallocatedCaseRegistration>,
   val inactiveRegistrations: List<UnallocatedCaseRegistration>,
-  val rosh: UnallocatedCaseRosh?,
```

Comment:
How are you intending to swap over to the new version? Without backwards compatibility there will have to be a bit of down time - albeit a few mins probably.

Reply:
The new implementation is a different endpoint. The old one is still there and unchanged. It is unwieldy and probably not very clear from this change, but basically I have renamed all the existing classes with the suffix of Deprecated. We can then switch the UI to the new endpoint and delete the Deprecated classes and controller endpoint. That will save coming up with a name like risksNew
On the endpoint itself, I was thinking to push another change after the UI has moved over, making the new endpoint available on `/risks` as well as `/risksNew`, switching the UI to that endpoint and then deleting `/risksNew` just for tidiness. It was either that or put a version somewhere in the URL or header, neither of which is particularly fancied.
It occurs to me now that I could have added a new Rosh object with a different name to the existing endpoint.

Reply:
Taken a new approach for no downtime